docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,615 @@
+---
+title: "Use ML and a webcam to entertain kids with bedtime songs"
+linkTitle: "Bedtime Songs Bot"
+weight: 60
+type: "docs"
+tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
+description: "Create a robot babysitter with a webcam and machine learning."
+image: "/tutorials/bedtime-songs-bot/bedtime-songs-bot.png"
+imageAlt: "Tess holds up brightly colored puzzle pieces in front of the camera of a Macbook laptop."
```

Comment:
```suggestion
webmSrc: "/tutorials/img/bedtime-songs-bot/robot_babysitter.webm"
mp4Src: "/tutorials/img/bedtime-songs-bot/robot_babysitter.mp4"
videoAlt: "A demonstration of the bedtime songs bot is taking place in an office. Tess holds up brightly colored puzzle pieces in front of the camera of a Macbook laptop. As the webcam on the laptop recognizes the puzzle pieces, different songs start to play on the speakers of the computer."
```

^ no reason not to use the full vid I think.

Reply:
There's currently a bug with how the card displays - the code only works if you have a valid cost. I'll fix this in a separate PR

Reply:
https://github.com/viamrobotics/docs/pull/1600

Reply:
@npentrel Got it, thanks so much!

Reply:
done